src/gui/render/graphics_data/individual_int.hpp

```diff
@@ -57,6 +57,7 @@ class IndividualInt {
 
  public:
     inline IndividualInt(const IndividualInt& other) = delete;
+    inline IndividualInt(const IndividualInt&& other) = delete;
```

Comment:
Why is this deleted?

Reply:
It basically holds pointers. Anything that holds pointers should not be copied.

Reply:
I'm not sure you understand what a **move** constructor is.

Reply:
```suggestion
    // Move-only
    IndividualInt(const IndividualInt& other) = delete;
    IndividualInt& operator=(const IndividualInt& other) = delete;
    
    IndividualInt(IndividualInt&& other) = default;
    IndividualInt& operator=(IndividualInt&& other) = default;
```

Reply:
Imagine, if you will, an object. Object A has a pointer to object B. There is no other way to create B. It only exists on the heap. The deconstructor for A deletes B from the heap.

Notice how coping A doesn't make sense. If I make a copy of an A, then delete one of the two copies the other object no longer works.

Reply:
Imagine, if you will, an object. Object A has a pointer to object H. There is no other way to create H. It only exists on the heap. The deconstructor for A deletes H from the heap.

Notice how moving A to B make sense. The data from A is transferred to B, meaning that B is now managing H, while A is left in a "valid, but undefined state", aka containing a nullptr.